src/routes/usersRouter.js

```diff
@@ -0,0 +1,19 @@
+const express = require('express');
+const router = express.Router();
+
+const {
+  addUser,
+  getUser,
+  updateUser,
+  deleteUser,
+  getRecords,
+  deleteRecords,
+} = require('../controller/userController');
+
+//users == base path
+router.route('/signup').post(addUser);
+router.route('/login').get(getUser);
```

Comment:
Hmm, it looks weird that our route for these two functions are going to be `/users/signup` and `/users/login`. Could you think of some reason we want those endpoints after `/users`?

Reply:
yes, we have a login page, and a signup page. users/signup was mentioned in the exercise description, thats why I build it like that

Reply:
Do you want me to change it ? that we have /signup and /login ?

Reply:
Ah, true, that is true. Hmm, I am not sure, it looks weird to have an endpoint called like that.

Reply:
Because we would be making calls to `/users/` ... Maybe there is a reason but I cannot see it. Can you think of some use case for doing like that?

Reply:
It makes not really sense, i just did it because of the exercise. wel, What do you think about we call the main rout just "/ " oder "/home" dann nutzen wir  "home/login" oder 'login' und "signup" oder "home/signup" ?

Reply:
I would call it just direct '`/login` and `/signup` then the routes in the backend match the routes in the front end too.